updates to readme

ayanangshu opened the pull request.

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Add files via upload

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Add files via upload

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Add files via upload

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Add files via upload

ayanangshu committed:
Add files via upload

ayanangshu committed:
Add files via upload

ayanangshu committed:
Update README.md

ayanangshu committed:
Add files via upload

ayanangshu committed:
Add files via upload

ayanangshu committed:
Update README.md

ayanangshu committed:
Add files via upload

ayanangshu committed:
Update README.md

ayanangshu committed:
Create images

ayanangshu committed:
Added an _images file

Added _images file to add images to readme

ayanangshu committed:
Update README.md

ayanangshu committed:
Remove the _images folder

borisd13 commented:
Looks great! Can you please:
- put all images together in the same folder "imgs"
- keep original readme from udacity at the bottom of the file so people know how to install/launch the project

ayanangshu committed:
Merge branch 'master' of https://github.com/ayanangshu/CarND-Capstone

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu committed:
Update README.md

ayanangshu commented:
I did the following changes:
1. Deleted _images folder
2. Put all the images used in readme in the existing imgs folder along with the images that were already existing in that folder
3. Changed the hyperlinks of the images to point to the images in the imgs folder
4. Added the original readme from Udacity

ayanangshu committed:
deleted the file images

ayanangshu committed:
Merge branch 'master' of https://github.com/ayanangshu/CarND-Capstone

ayanangshu committed:
Update README.md

ayanangshu committed:
Updated Hyperlinks

Updated the hyperlinks for images

borisdayma committed:
Merge branch 'master' into master

borisdayma committed:
Merge branch 'master' into master
